CONTRIBUTING.md

```diff
@@ -83,13 +101,50 @@ npm run dev:pristine
 npm run dev:format
 ```
 
+## Coding conventions
+
+1. We recommend small, single-purpose components. We recommend breaking up large components into smaller, more cohesive components. Smaller components are easier to test because you can isolate functionality to ensure that components are immune to changes in other components. In general, larger components lead to larger tests that are more difficult to write and more difficult to understand.
+1. Use the `Sky` prefix when naming all classes, directives, services, components, etc. The prefix indicates to other contributors that items are owned by SKY UX and not a third-party library. It also prevents potential class-name clashes with other libraries. Keep in mind that while we generally use the uppercase `Sky` prefix, we also use the `sky-` prefix in some cases, such as selector properties in components.
+
+## Test harnesses
```

Comment:
Same question as above about whether I need to replace the "Test fixtures" section in the current docs (https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#test-fixtures) with this section.

Reply:
Same answer - that page is going away.